package.json

```
@@ -40,63 +40,82 @@
     "broccoli-rollup": "^4.1.1",
-    "ember-cli-babel": "^7.12.0",
-    "ember-cli-htmlbars": "^3.0.1",
-    "ember-cli-version-checker": "^3.1.3",
-    "ember-compatibility-helpers": "^1.2.1",
+    "ember-cli-babel": "^7.23.0",
+    "ember-cli-htmlbars": "^5.7.1",
+    "ember-cli-version-checker": "^5.1.2",
+    "ember-compatibility-helpers": "^1.2.5",
     "ember-raf-scheduler": "0.2.0"
   },
   "devDependencies": {
-    "@ember/optional-features": "^0.7.0",
+    "@ember/optional-features": "^2.0.0",
+    "@ember/string": "^3.0.0",
+    "@ember/test-helpers": "^2.1.4",
+    "@glimmer/component": "^1.0.3",
+    "@glimmer/tracking": "^1.0.3",
+    "babel-eslint": "^10.1.0",
     "bootstrap": "~3.3.5",
     "broccoli-asset-rev": "^3.0.0",
-    "ember-cli": "~3.12.0",
-    "ember-cli-app-version": "^3.2.0",
-    "ember-cli-dependency-checker": "^3.1.0",
-    "ember-cli-eslint": "^5.1.0",
+    "ember-auto-import": "^1.10.1",
+    "ember-cli": "~3.24.0",
+    "ember-cli-app-version": "^5.0.0",
+    "ember-cli-dependency-checker": "^3.2.0",
+    "ember-cli-deprecation-workflow": "^2.0.0",
     "ember-cli-fastboot": "^2.1.3",
     "ember-cli-github-pages": "^0.2.2",
-    "ember-cli-htmlbars-inline-precompile": "^2.1.0",
-    "ember-cli-inject-live-reload": "^2.0.1",
+    "ember-cli-inject-live-reload": "^2.0.2",
     "ember-cli-sri": "^2.1.1",
-    "ember-cli-template-lint": "^1.0.0-beta.1",
-    "ember-cli-uglify": "^3.0.0",
+    "ember-cli-terser": "^4.0.1",
     "ember-code-snippet": "^2.4.1",
-    "ember-data": "3.11.5",
+    "ember-data": "~3.26.0",
     "ember-disable-prototype-extensions": "^1.1.3",
-    "ember-export-application-global": "^2.0.0",
-    "ember-load-initializers": "^2.0.0",
+    "ember-export-application-global": "^2.0.1",
+    "ember-load-initializers": "^2.1.2",
+    "ember-maybe-import-regenerator": "^1.0.0",
     "ember-native-dom-helpers": "^0.7.0",
-    "ember-perf-timeline": "^1.2.1",
-    "ember-qunit": "^4.5.1",
-    "ember-resolver": "^5.0.1",
-    "ember-source": "~3.12.0",
-    "ember-source-channel-url": "^2.0.1",
-    "ember-try": "^1.2.1",
-    "eslint-plugin-ember": "^6.7.1",
-    "eslint-plugin-node": "^9.1.0",
+    "ember-perf-timeline": "^2.1.0",
+    "ember-qunit": "^4.6.0",
+    "ember-resolver": "^8.0.2",
+    "ember-source": "~3.24.0",
+    "ember-source-channel-url": "^3.0.0",
+    "ember-template-lint": "^2.15.0",
+    "ember-try": "^1.4.0",
+    "eslint": "^7.17.0",
+    "eslint-config-prettier": "^7.1.0",
+    "eslint-plugin-ember": "^10.1.1",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^3.3.1",
     "loader.js": "^4.7.0",
     "npm-run-all": "^4.1.5",
-    "qunit-dom": "^0.9.0",
+    "prettier": "^2.2.1",
+    "qunit-dom": "^1.6.0",
     "release-it": "^14.2.1",
     "release-it-lerna-changelog": "^3.1.0"
   },
+  "resolutions": {
+    "**/ember-cli-babel": "^7.26.6"
+  },
   "engines": {
-    "node": ">= 10.*"
+    "node": "10.* || >= 12"
```

```suggestion
    "node": "12.* || >= 14"
```

Since you are only testing Node 12 on CI (and Node 10 is EOL anyway)

---

This will be included with future ember-cli blueprints updates